hedera-node/hedera-app/src/test/java/com/hedera/node/app/grpc/impl/netty/GrpcServiceBuilderTest.java

```diff
@@ -104,7 +104,7 @@ void queryIsBlank(final String value) {
     @Test
     @DisplayName("The build method will return a ServiceDescriptor")
     void serviceDescriptorIsNotNullOnNoopBuilder() {
-        assertNotNull(builder.build(metrics));
+        assertNotNull(builder.build(metrics), "Object is Null!");
```

Comment:
We agreed to use `AssertJ` for assertions. While this is not consistently enforced because we never cleaned up existing code, new code should not use other libraries.

Feel free to clean up the other assertions in this class.

Reply:
Thank you for this comment, I was not aware that we have agreed on AssertJ, I changed it for the new tests.